src/ObjectGraph.es6.js

```diff
@@ -96,8 +106,10 @@ ObjectGraph.prototype.blacklistedProperties = [
 
 ObjectGraph.prototype.initLazyData = function() {
   stdlib.memo(this, 'invTypes', () => {
-    let invTypes = remap['a:b=>b:[a]'](this.types);
-    _.forOwn(invTypes, (v, k, o) => o[k] = v[0]);
+    var invTypes = {};
+    Object.keys(this.types).map(type => {
```

Comment:
Here and elsewhere: Have you benchmarked this? `for ( var key in o ) { if ( o.hasOwnProperty(key) ) ... }` generally performs faster, but if the difference is not appreciable in practice, I'm fine with this.